nautobot_golden_config/filters.py

```diff
@@ -22,37 +22,37 @@ class GoldenConfigFilter(CreatedUpdatedFilterSet):
     )
     tenant_group_id = TreeNodeMultipleChoiceFilter(
         queryset=TenantGroup.objects.all(),
-        field_name="device__tenant__group",
+        field_name="tenant__group",
```

Comment:
Was this actually needed? 

I also fear this may affect the filtering on this view. 
![image](https://user-images.githubusercontent.com/9260483/131200071-396761b7-771e-4104-a5e9-c330694ae29c.png)

Reply:
Found the filter was broken, but when I was debugging I went to the demo.nautobot.com instance and it's broken there as well with the same error.  

```
A database programming error was detected while processing this request. Common causes include the following:

 Database migrations missing - When upgrading to a new Nautobot release, the upgrade script must be run to apply any new database migrations. You can run migrations manually by executing nautobot-server migrate from the command line.

 Unsupported PostgreSQL version - Ensure that PostgreSQL version 9.6 or higher is in use. You can check this by connecting to the database using Nautobot's credentials and issuing a query for SELECT VERSION().
```

Reply:
I don't see any migrations that line up, is it possible we have a missing migration?

Reply:
Was this resolved? Should not be an issue.

Reply:
Not resolved, it doesn't work in the demo instance either feel like this was broken in the current release as well?

Reply:
let's sync